tests/acceptance/book-appointment-test.js

```diff
@@ -139,37 +139,41 @@ test("request purpose page should not redirect if incomplete form", function(ass
   });
 });
 
-test("request purpose page on completely filled should redirect to client information if For client benificiary is selected", function(assert){
-  orderPurpose2 = {id: 1, purpose_id: 2, order_id: order.id, designation_id: 1};
-  assert.expect(2);
-  visit(requestPurposeUrl);
-  andThen(function(){
-    click(".for-client");
-  });
-  andThen(function(){
-    click("#people-count");
-  });
-  andThen(function(){
-    fillIn("#people-count", "2");
-  });
-  andThen(function(){
-    fillIn("#description", "test test test");
-  });
-  andThen(function(){
-    mocks.push(
-      $.mockjax({url: '/api/v1/order*', type: 'POST', status: 201,responseText: {
-        order: order.toJSON({includeId: true}),
-        orders_purposes: [orderPurpose2],
-        user: user.toJSON({includeId: true})}
-      })
-    );
-    click('#request-submit');
-    andThen(function(){
-      assert.equal(currentURL(), clientInfoUrl);
-      assert.equal(Ember.$('.title').text().trim(), "Client Information");
-    });
-  });
-});
+// test("request purpose page on completely filled should redirect to client information if For client benificiary is selected", function(assert){
```

Comment:
Why this test case was commented out?

Reply:
@Darshan-Chauhan, i'm working on it in a seperate branch, that's why